BackEnd/package.json

```diff
@@ -45,7 +45,6 @@
     "@types/node": "^20.2.5",
     "@types/uuid": "^8.3.4",
     "@types/yargs": "^17.0.11",
-    "chartjs-node-canvas": "^4.1.6",
```

Comment:
Turns out this isnt compatible with nodejs 20, thankfully we dont use this dependency anywhere!

Reply:
Oh yeah. I was planning to use it to generate the graphs on the backend for export and download.